README.md

```diff
@@ -184,6 +184,7 @@ directory.
       * `wrap` - to wrap the extension point UI
       * `before` - to place the component before the extension point
       * `after` - to place the component after the extension point
+  * **config** (optional) - plugin configuration
```

Comment:
Let's add this field to the example above. And to my mind we should write available type. Like:
```
* **config** (optional) `Object` - ...
```

You can also describe other fields in the same way.